packages/libretro/flycast/package.mk

```diff
@@ -19,7 +19,7 @@
 ################################################################################
 
 PKG_NAME="flycast"
-PKG_VERSION="ae670ea"
+PKG_VERSION="e9bc945"
 PKG_ARCH="any"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/libretro/flycast"
```

Comment:
We should  try to take opportunity to switch to [upstream](https://github.com/flyinghead/flycast) as it will fix [this issue](https://github.com/libretro/Lakka-LibreELEC/issues/1560).

@ToKe79 Arm compilation has been fixed with [bf87f7b](https://github.com/flyinghead/flycast/commit/bf87f7ba8403ee394233438b68177e0caf18ab1d).

Do you validate this package ?

```
################################################################################
#      This file is part of OpenELEC - http://www.openelec.tv
#      Copyright (C) 2009-2012 Stephan Raue (stephan@openelec.tv)
#
#  This Program is free software; you can redistribute it and/or modify
#  it under the terms of the GNU General Public License as published by
#  the Free Software Foundation; either version 2, or (at your option)
#  any later version.
#
#  This Program is distributed in the hope that it will be useful,
#  but WITHOUT ANY WARRANTY; without even the implied warranty of
#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
#  GNU General Public License for more details.
#
#  You should have received a copy of the GNU General Public License
#  along with OpenELEC.tv; see the file COPYING.  If not, write to
#  the Free Software Foundation, 51 Franklin Street, Suite 500, Boston, MA 02110, USA.
#  http://www.gnu.org/copyleft/gpl.html
################################################################################

PKG_NAME="flycast"
PKG_VERSION="bf87f7b"
PKG_ARCH="any"
PKG_LICENSE="GPLv2"
PKG_SITE="https://github.com/flyinghead/flycast"
PKG_URL="$PKG_SITE.git"
PKG_DEPENDS_TARGET="toolchain"
PKG_PRIORITY="optional"
PKG_SECTION="libretro"
PKG_SHORTDESC="Flycast is a multiplatform Sega Dreamcast emulator"
PKG_LONGDESC="Flycast is a multiplatform Sega Dreamcast emulator"
PKG_BUILD_FLAGS="-lto"
PKG_TOOLCHAIN="cmake"

PKG_IS_ADDON="no"
PKG_AUTORECONF="no"

if [ "$OPENGL_SUPPORT" = yes ]; then
  PKG_DEPENDS_TARGET+=" $OPENGL"
fi

if [ "$OPENGLES_SUPPORT" = yes ]; then
  PKG_DEPENDS_TARGET+=" $OPENGLES"
fi

PKG_CMAKE_OPTS_TARGET="-DLIBRETRO=ON \
		      -DUSE_OPENMP=OFF \ 
		      -DUSE_GLES=ON \ 
                       -DCMAKE_BUILD_TYPE=Release \
                       --target flycast_libretro"

if [ "$OPENGL_SUPPORT" = no -a "$OPENGLES_SUPPORT" = yes ]; then
  PKG_CMAKE_OPTS_TARGET+=" -DUSING_GLES=ON"
fi

if [ "$VULKAN_SUPPORT" = yes ]; then
  PKG_DEPENDS_TARGET+=" ${VULKAN}"
  PKG_CMAKE_OPTS_TARGET+=" -DVULKAN=ON"
fi

if [ "$TARGET_ARCH" = "arm" ]; then
  PKG_CMAKE_OPTS_TARGET+=" -DARMV7=ON"
elif [ "$TARGET_ARCH" = "aarch64" ]; then
  PKG_CMAKE_OPTS_TARGET+=" -DARM64=ON"
fi

pre_make_target() {
  find $PKG_BUILD -name flags.make -exec sed -i "s:isystem :I:g" \{} \;
  find $PKG_BUILD -name build.ninja -exec sed -i "s:isystem :I:g" \{} \;
}

makeinstall_target() {
  mkdir -p $INSTALL/usr/lib/libretro
  cp flycast_libretro.so $INSTALL/usr/lib/libretro/
}
```

Thank you.

Reply:
Sounds good. Made the commit.

Reply:
Sorry I made a typo 

```
PKG_CMAKE_OPTS_TARGET="-DLIBRETRO=ON \
		      -DUSE_OPENMP=OFF \ 
		      -DUSE_GLES=ON \ 
                       -DCMAKE_BUILD_TYPE=Release \
                       --target flycast_libretro"
```
should be (without `-DUSE_GLES=ON \`)
```
PKG_CMAKE_OPTS_TARGET="-DLIBRETRO=ON \
		      -DUSE_OPENMP=OFF \ 
                       -DCMAKE_BUILD_TYPE=Release \
                       --target flycast_libretro"
```